src/app/(auth)/editprofile.tsx

```diff
@@ -0,0 +1,137 @@
+import Icon from "@/src/components/Icon";
+import { Text, View } from "@/src/components/Themed";
+import DefaultColors from "@/src/constants/DefaultColors";
+import { fetchSingleUserThumbnail } from "@/src/lib/getRandomProfilePics";
+import { resetNavigationStack } from "@/src/lib/resetNavigationStack";
+import {
+  IconTrash,
+  IconCirclePlus,
+  IconX,
+  IconCheck,
+} from "@tabler/icons-react-native";
+import { useColorScheme } from "nativewind";
+import { useEffect, useState } from "react";
+import { Pressable, StatusBar } from "react-native";
+import { Link } from "expo-router";
+import {
+  Image,
+  Platform,
+  SafeAreaView,
+  TextInput,
+  TouchableOpacity,
+} from "react-native";
+
+export default function Createprofile() {
+  const { colorScheme } = useColorScheme();
+  const [displayName, setDisplayName] = useState("");
+  const [username, setUsername] = useState("");
+  const [pfp, setPfp] = useState();
+
+  useEffect(() => {
+    fetchSingleUserThumbnail().then((user) => {
+      setPfp(user.imgurl);
+    });
+  }, []);
+
+  return (
+    <SafeAreaView
+      className={`flex flex-1 flex-col items-center bg-white dark:bg-stone-800`}
+      style={{
+        paddingTop:
+          Platform.OS === "android"
+            ? Math.max(StatusBar.currentHeight || 0, 34)
+            : 0,
+      }}
+    >
+      {/* Form */}
+      <View className="flex w-screen flex-col px-5">
```

Comment:
Don't copy and paste everything; create a new component and reuse it in this page and the create profile page